Sources/AppStoreConnectCLI/Services/Operations/GetAppsOperation.swift

```diff
@@ -20,20 +24,20 @@ struct GetAppsOperation: APIOperation {
         }
     }
 
-    private let options: GetAppsOptions
+    private let options: Options
 
-    init(options: GetAppsOptions) {
+    init(options: Options) {
         self.options = options
     }
 
-    func execute(
-        with requestor: EndpointRequestor
-    ) -> AnyPublisher<[AppStoreConnect_Swift_SDK.App], Error> {
+    typealias App = AppStoreConnect_Swift_SDK.App
+
+    func execute(with requestor: EndpointRequestor) -> AnyPublisher<[App], Error> {
         let bundleIds = options.bundleIds
         let endpoint = APIEndpoint.apps(filters: [.bundleId(bundleIds)])
 
         return requestor.request(endpoint)
-            .tryMap { (response: AppsResponse) throws -> [AppStoreConnect_Swift_SDK.App] in
+            .tryMap { (response: AppsResponse) throws -> [App] in
                 guard !response.data.isEmpty else {
                     throw GetAppIdsError.couldntFindAnyAppsMatching(bundleIds: bundleIds)
                 }
```

Comment:
@adamjcampbell Just found a small bug below this line in a commit some days before.
```
guard responseBundleIds == bundleIds else {
    let nonExistentBundleIds = responseBundleIds.subtracting(bundleIds)  
    ...

}
```
`let nonExistentBundleIds = responseBundleIds.subtracting(bundleIds)`
here should be -> 
`et nonExistentBundleIds = bundleIds.subtracting(responseBundleIds)`